src/rp2_common/pico_audio_pwm/audio_pwm.c

```diff
@@ -78,8 +78,8 @@ const audio_pwm_channel_config_t default_right_channel_config =
         {
                 .core = {
                         .base_pin = PICO_AUDIO_PWM_R_PIN,
-                        .pio_sm = 1,
```

Comment:
Hmmm, I've only just noticed that this used to set `.pio_sm` to 1, but `PICO_AUDIO_PWM_R_PIO_SM` is now set to default to 0 ? :eyes: 
(but I've never tried running this code, so I dunno if that'll cause problems or not)

Reply:
Yes, that would in fact cause an issue... I'll open up a revert pr..